objectstore.go

```diff
@@ -13,49 +15,40 @@ import (
 )
 
 const (
-	accessGrant = "accessGrant"
+	accessGrant        = "accessGrant"
+	linksharingBaseURL = "https://link.tardigradeshare.io"
```

Comment:
We need to make it configurable (but with the above default value), so for tests we can use a URL for locally running linksharing service.